spec/acceptance/sensu_plugin_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper_acceptance'
+
+describe 'sensu_plugin', if: RSpec.configuration.sensu_full do
+  agent = hosts_as('sensu_agent')[0]
+  before do
+    if fact_on(agent, 'operatingsystem') == 'Debian'
+      skip("TODO: package is missing on Debian - See https://github.com/sensu/sensu-go/issues/2757")
```

Comment:
Should probably just link to https://github.com/sensu/sensu-plugins-omnibus/issues/3

Reply:
Updated